rezervo/schemas/schedule.py

```diff
@@ -79,6 +79,12 @@ class UserSession(OrmBase):
     class_data: SessionRezervoClass
 
 
+class UserAgendaClass(BaseModel):
+    chain: ChainIdentifier
+    status: SessionState
+    class_data: SessionRezervoClass
```

Comment:
Can we maybe define `UserSession` as an extension of a smaller model containing `chain`, `status`, `class_data` (e.g. `BaseUserSession`) and use this smaller model instead of UserAgendaClass.

Reply:
Yeah, I realise now that there is no reason to introduce something named AgendaClass when this is basically a session. Refactored the frontend and backend now: https://github.com/mathiazom/rezervo/pull/46/commits/0fb4fd45d152cd507fd5c34e950f7c20d5f6f495